src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -281,7 +281,7 @@ public boolean delete() {
             Optional<ButtonType> buttonType = dialogService.showCustomButtonDialogAndWait(AlertType.INFORMATION,
                     Localization.lang("Delete '%0'", file.get().toString()),
                     Localization.lang("Delete the selected file permanently from disk, or just remove the file from the entry? Pressing Delete will delete the file permanently from disk."),
-                    deleteFromEntry, removeFromEntry, ButtonType.CANCEL);
+                    removeFromEntry, deleteFromEntry, ButtonType.CANCEL);
```

Comment:
This change is fine, but could you please add [ButtonData.Yes](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/ButtonBar.ButtonData.html#YES) as a second constructor argument for the `removeFromEntry` button. With this the buttons are automatically ordered correctly according to the current OS specifications.

Reply:
Sure, not a problem. Would you like to have the change rebased into the first commit, or is a new one on top of the current two acceptable?

Reply:
Just add them in a new commit. We usually squash all commits on merge